tests/contrib/psycopg/test_psycopg_async.py

```diff
@@ -331,19 +331,6 @@ async def test_connection_context_execute(self):
             assert len(rows) == 1, rows
             assert rows[0][0] == "one"
 
-    @mark_asyncio
-    async def test_cursor_context_execute(self):
-        """Checks whether cursor context manager works as normal."""
-
-        query = SQL("""select 'one' as x""")
-        async with (await psycopg.AsyncConnection.connect(**POSTGRES_CONFIG)).cursor() as cur:
-            breakpoint()
```

Comment:
Does removing the `breakpoint()` fix the failing test? I don't see how the breakpoint is relevant to this test...

Reply:
good catch - TIL that you can use `breakpoint` instead of `import pdb;pdb.set_trace()`